src/types.ts

```diff
@@ -242,3 +242,13 @@ export const contentContextTypes: ValidationMap<any> = {
 export interface WithContextTypes<IJ> {
   context: IJ;
 }
+
+export enum TypeOf {
```

Comment:
If we're only using this in a single file would it be easier to keep this in isObject.ts too?

Do you see us remembering to use this enum in the future or will we end up sticking to plain JS and doing things like `typeof x === 'boolean'`. My gut is thinking we'll forget that this exists and we might as well do `(type === 'object' || type === 'function');`.

Reply:
I used it in a few different utility methods I burned from lodash. That's a good question, we'll probably wants others opinion on that. cc/ @daniedleroux @danrosenthal

Reply:
Cool, I wasn't aware of that.

Reply:
I don't have strong feelings. If others find it useful, I don't have any problem with it